geomstats/geometry/grassmannian.py

```diff
@@ -55,3 +59,22 @@ def __init__(self, n, p):
                 dimension=dimension,
                 signature=(dimension, 0, 0))
         self.embedding_metric = EuclideanMetric(n * p)
+
+    def exp(self, v, p):
```

Comment:
v -> tangent_vector
p -> base_point

As a rule of thumb, we avoid single letter as variable names, in order to be able to use automatic find and replace